lib/definitions/cloud-build-service.d.ts

```diff
@@ -281,6 +286,10 @@ interface IIOSBuildData extends IBuildForDevice {
 interface ICloudBuildOutputDirectoryOptions extends IOutputDirectoryOptions {
 }
 
+interface ICloudBuildOutputOptions extends IOutputDirectoryOptions {
+	extension?: string;
```

Comment:
There is no need to create new interface for this. Add `extension?: string` in the `IOutputDirectoryOptions`

Reply:
Update: IOutputDirectoryOptions is in NS CLI code, so I do not think this is a good idea. Renamed it to ICloudOperationOutputOptions.